swagger-ui/swagger.yaml

```diff
@@ -669,9 +702,18 @@
                 surplus_auction_threshold:
                   type: string
                   example: '1000000000'
+                surplus_auction_lot:
+                  type: string
+                  example: '10000000'
                 debt_auction_threshold:
                   type: string
                   example: '1000000000'
+                surplus_auction_lot:
+                  type: string
+                  example: '10000000'
+                savings_distribution_frequency:
+                  type: string
+                  example: '60000000'
                 circuit_breaker:
                   type: boolean
                   example: false
```

Comment:
Can't highlight the right line here, but
```
/cdp/cdps/cdp/{owner}/{denom}
```

should become
```
/cdp/cdps/cdp/{owner}/{collateral-type}
```

Reply:
Lines 424, 478, 532, 584, 722, 749, 779, 815, 1397

Reply:
https://github.com/Kava-Labs/kava/pull/653/commits/550065a767bc10dd230a8ee7f9a15a0b6742d1a3